### ISSUE TITLE
Make a developer APB to help develop the broker & catalog

### ISSUE THREAD

**Issue by rthallisey** [fusor/apb-examples#26]
It would be nice to create an APB that could be used by developers working on the broker & catalog. The apb would be very simple and can perform [provision, deprovision, bind, unbind] in a flexible and debuggable manner. It would also serve as an example template for array of APBs. A perfect example of this is an apb that has a pinch of every feature we're adding into an apb, but as all hard coded values. The hello-world-apb currently is useful in that it's simple, but it doesn't have bind.

**Comment by tsanders-rh** [comment]
Agreed.  Does adding bind support to hello-world-apb make sense?

-Todd

**Comment by rthallisey** [comment]
Yes, that would help.  Also, adding some simple parameters, meta data, ect ... so that it has every possible thing you can do with an APB hard coded in there.

For example as a developer, I can use this as a template APB for my new App that I'm building. Or I can use it for testing a new feature for the broker without having to use an APB I don't understand.  Also, I think an APB like this would be useful for gating/CI in the broker down the road.
